src/components/MySpace/MySpace.tsx

```diff
@@ -53,7 +53,7 @@ const MySpace = ({ bookmarks }: { bookmarks: BookmarkRecords }) => {
             data.collections.map((collection) => (
               <Grid
                 key={`collection_${collection._id}`}
-                tablet={{ col: 6 }}
+                tabletLg={{ col: 6 }}
```

Comment:
so first, apologies, I realize the syntax for the grid classes is kind of confusing. This change is telling the layout to go from 1 item/row > 2 items/row at `tabletLg`, which I think we do want so the edit title form doesn't wrap. However I think we _also_ want to go from 2 items/row > 3 items/row at a larger size, so we also need:
```
- desktop={{ col: 4 }}
+ desktopLg={{ col: 4 }}
```

However both the `tabletLg, desktopLg` breakpoints are actually [turned off in USWDS by default](https://designsystem.digital.gov/utilities/layout-grid/) so these changes aren't having any effect. to fix, we need to turn them on in our `styles/_settings.scss` file by adding this:

```
$theme-utility-breakpoints: (
  'card': false,  // 160px
  'card-lg': false,  // 240px
  'mobile': false,  // 320px
  'mobile-lg': true,  // 480px
  'tablet': true,  // 640px
  'tablet-lg': true,  // 800px
  'desktop': true,  // 1024px
  'desktop-lg': true,  // 1200px
  'widescreen': false,  // 1400px
);
```

Reply:
thank you so much!!